bears/typescript/TSLintBear.py

```diff
@@ -26,16 +52,142 @@ class TSLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         tslint_config: path="", rules_dir: path=""):
+                         tslint_config: path="",
+                         rules_dir: path="",
+                         directive_selectors_naming_convention: str="camel",
+                         component_selectors_naming_convention: str="hyphen",
+                         directive_selectors_type: list=[True, "attribute"],
+                         component_selectors_type: list=[True, "element"],
+                         directive_selectors_prefix: list=[True, "sg"],
+                         component_selectors_prefix: list=[True, "sg"],
```

Comment:
I feel like I already reviewed this.... why is there this weird lists as a default? It's truly not obvious for the user what this means and defaults should be obvious.